From 123d41d1dd202212371a07c480ae27ba79e347f9 Mon Sep 17 00:00:00 2001
From: Feng Xiao <xf@rock-chips.com>
Date: Tue, 8 Mar 2016 16:11:05 +0800
Subject: [PATCH] clk: rockchip: add video noc clk to the list of rk3366
 critical clocks

The clocks of VPU NOC and RKVEDC NOC interact with each other.
If one of VPU and RKVDEC is working, they all must be opened.

Change-Id: I966df107ae72fbbb99f1e660a79bfd07476e8539
Signed-off-by: Feng Xiao <xf@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3366.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/clk/rockchip/clk-rk3366.c b/drivers/clk/rockchip/clk-rk3366.c
index 4953ba01b2cf..8d631e912097 100644
--- a/drivers/clk/rockchip/clk-rk3366.c
+++ b/drivers/clk/rockchip/clk-rk3366.c
@@ -778,8 +778,12 @@ static const char *const rk3366_critical_clocks[] __initconst = {
 	"aclk_bus",
 	"aclk_peri0",
 	"aclk_peri1",
+	"aclk_video_noc",
+	"aclk_rkvdec_noc",
 	"hclk_peri0",
 	"hclk_peri1",
+	"hclk_video_noc",
+	"hclk_rkvdec_noc",
 	"pclk_peri0",
 	"pclk_peri1",
 	"pclk_rk_pwm",
-- 
2.35.3

